ihmc-footstep-planning/src/main/generated-java/us/ihmc/footstepPlanning/AStarBodyPathPlannerParametersReadOnly.java

```diff
@@ -372,4 +373,67 @@ default double getSmootherGradientThresholdToTerminate()
    {
       return get(smootherGradientThresholdToTerminate);
    }
+
+   default AStarBodyPathPlannerParametersPacket getAsPacket()
```

Comment:
@PotatoPeeler3000 Do we need this? There was a tools method created to do this, investigate

Reply:
Well we can for logging the data, but then we don't need to have both the tools method and this...